zebra-consensus/src/transaction/tests.rs

```diff
@@ -160,3 +164,33 @@ fn v5_coinbase_transaction_with_enable_spends_flag_fails_validation() {
         Err(TransactionError::CoinbaseHasEnableSpendsOrchard)
     );
 }
+
+#[tokio::test]
+async fn v5_transaction_is_rejected_before_nu5_activation() {
+    let network = Network::Mainnet;
+    let blocks = zebra_test::vectors::MAINNET_BLOCKS.iter();
```

Comment:
Let's run this test for both mainnet and testnet, because testnet will get real v5 blocks in its test vectors before mainnet.